auth/resources/authserver.properties.sample

```diff
@@ -26,14 +26,18 @@ httpsPort=28051
 defaultHost=0.0.0.0
 logFilePath=/var/log/seagate/auth/server
 samlMetadataFileName=saml-metadata.xml
+# Max limit of entries for account, iamuser and ldapsearch
+# operations can be configured using below three paramters.
+maxAccountLimit=1000
```

Comment:
Is 1000 ok or should it be 1001 ?

Reply:
yes. t will be 1000 since this is client configuration parameters which will represent client account limit only . 
For internal accounts we are using another method : AuthServerConfig.getS3InternalAccounts().size(); 
this method will return internal account count which we are considering while checking ldap entries